<?xml version="1.0"?>
<launch>
  <!-- <master auto="start"/>  -->
<!--   <param name="use_sim_time" value="true" /> -->
  <param name="simple_move_base_cfgfile" value="$(find patrol_robot)/param/simple_move_base_params.yaml" />

  <!--   map_server -->
  <node pkg="map_server" type="map_server" name="map_server" args="$(find patrol_robot)/map/mapv0.2.yaml"/>

  <!--   if sim, process vlp16 in sim -->
  <include file="$(find patrol_robot)/launch/handle_sim_vlp16.launch" />

  <!--   als -->
  <include file="$(find als_ros)/launch/mcl_patrol_robot.launch" />

  <!--   simple_move_base server -->
  <node pkg="patrol_robot" type="simple_move_base_server_fsm.py" name="simple_move_base_server" output="screen" />

  <!--   simple_move_base client -->
  <node pkg="patrol_robot" type="simple_move_base_client.py" name="simple_move_base_client" output="screen" />

  <!--   rviz -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find patrol_robot)/rviz_cfg/task_nav.rviz"/>
</launch> 

